src/pages/VestingClaim.tsx

```diff
@@ -120,17 +119,17 @@ function VestingClaim(): ReactElement {
 
   return (
     <>
-      <div className={styles.container}>
-        <div className={styles.gradient}></div>
-        <div className={styles.logoWrapper}>
-          <div className={styles.logo}>
+      <div>
+        <div></div>
+        <div>
+          <div>
```

Comment:
can you get rid of these divs if they aren't essential to the building of this component?